meson.build

```diff
@@ -20,8 +20,9 @@ libzim_dep = dependency('libzim', version:'>=9.0.0', static:static_linkage)
 libzim_dep = dependency('libzim', version:'<10.0.0', static:static_linkage)
 kiwixlib_dep = dependency('kiwix', version:'>=13.0.0', static:static_linkage)
 kiwixlib_dep = dependency('kiwix', version:'<14.0.0', static:static_linkage)
+docopt_dep = dependency('docopt', static:static_linkage)
```

Comment:
I would split this commit ("Port kiwix-tools to docopt.") into three - one per tool.